docs/reference/release-notes/highlights.asciidoc

```diff
@@ -12,7 +12,7 @@ endif::[]
 // Add previous release to the list
 Other versions:
 
-{ref-bare}/8.6/release-highlights.html[8.6]
+//{ref-bare}/8.6/release-highlights.html[8.6]
```

Comment:
I wonder if the `|` in the line below would need to be removed (since it seems now to be at the top of this list).

I also wonder why this error happened in the first place. Possibly there were no 8.6 highlights and that's why?

Reply:
I pushed a commit to remove that first `|` and it looks good when I build locally.

Yeah, I would guess that having no 8.6 highlights is what caused the error.